garrysmod/lua/includes/modules/outline.lua

```diff
@@ -0,0 +1,120 @@
+module("outline",package.seeall)
+
+local List,ListSize = {},0
+local RenderEnt = NULL
+
+local CopyMat		= Material("pp/copy")
+local OutlineMat	= CreateMaterial("OutlineMat","UnlitGeneric",{["$ignorez"] = 1,["$alphatest"] = 1})
+local StoreTexture	= render.GetScreenEffectTexture(0)
+local DrawTexture	= render.GetScreenEffectTexture(1)
+
+local ENTS,COLOR,MODE = 1,2,3
+local MODE_NOTVISIBLE,MODE_VISIBLE = 1,2
```

Comment:
I would make these global and name them ``RENDER_MODE_*`` for usage outside of this file. You should also add a ``RENDER_MODE_BOTH`` and make it the default if a mode is not specified.

Reply:
maybe `OUTLINE_MODE_*` will be more correct

Reply:
Whoops that's what I meant